python/lsst/sims/catUtils/exampleCatalogDefinitions/phoSimCatalogExamples.py

```diff
@@ -290,15 +301,30 @@ class PhoSimCatalogZPoint(PhosimInputBase, PhoSimAstrometryGalaxies, EBVmixin):
     transformations = {'raPhoSim': np.degrees, 'decPhoSim': np.degrees}
 
 
+class PhoSimCatalogSN(PhoSimCatalogZPoint, FrozenSNCat, EBVmixin):
```

Comment:
Shouldn't PhoSimCatalogSN set `self.writeSedFile=True` by default?  Is there any case in which a user would want to write a PhoSimCatalog of SNe but not write out the SEDs?